eng/native/configurecompiler.cmake

```diff
@@ -85,6 +85,8 @@ if (MSVC)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /sourcelink:${CLR_SOURCELINK_FILE_PATH}")
   endif(EXISTS ${CLR_SOURCELINK_FILE_PATH})
 
+  add_linker_flag(/INCREMENTAL:NO) # Not compatible with /DEBUGTYPE
```

Comment:
Does this means `./build clr.jit` will always rebuild all the files? It is very worrying of so.

Why doesn't upstream hit incompatibility error?

Reply:
I think this only happens with `-msbuild`.  I think this is the same: https://github.com/dotnet/runtime/issues/64082